draft-ietf-regext-rdap-geofeed.md

```diff
@@ -43,13 +43,13 @@ the associated link objects included in responses.
 [@RFC8805] and [@!RFC9632] detail the IP geolocation feed (commonly known as 'geofeed') file format and associated
 access mechanisms. While [@!RFC9632] describes how a registry can make geofeed URLs available by way of a Routing
 Policy Specification Language (RPSL) [@RFC2622] service, the Regional Internet Registries (RIRs) have deployed
-Registration Data Access Protocol (RDAP) ([@RFC7480], [@RFC7481], [@!RFC9082], [@!RFC9083]) services as successors for
+Registration Data Access Protocol (RDAP) ([@RFC7480], [@RFC7481], [@!RFC9082], [@!RFC9083]) services as successors to
 RPSL for Internet number resource registrations, and maintaining feature parity between the two services supports client
 transition from RPSL to RDAP in this context. To that end, this document specifies how geofeed URLs can be accessed
 through RDAP. It defines a new RDAP extension, "geofeed1", for indicating that an RDAP server hosts geofeed URLs for its
 IP network objects, as well as a new media type and a new link relation type for the associated link objects.
 
-Fetching and making use of geofeed data is out of scope for the purposes of this document. See [@RFC8805] and
+Fetching and making use of geofeed data are out of scope for the purposes of this document. See [@RFC8805] and
```

Comment:
AIUI, "Fetching ... data" is a singular noun phrase (compare e.g. "Fetching geofeed data and making use of geofeed data"), so 'is' is the right verb here.  I'd prefer to leave this as-is and let the RFC editor change it if necessary.